### DISCUSSION TITLE
Migrate from Phin?

### DISCUSSION THREAD

**Discussion by dzmikhalchuk** [pactumjs/pactum#327] [Q&A]
Hey guys! Thanks for your job, I really like PactumJS. I saw that Phin is [deprecated](https://github.com/ethanent/phin/issues/91). Are you planning to migrate to another http solution?

**Comment by ASaiAnudeep** [comment]
Thanks for bringing this up. We will look into it.
cc: @leelaprasadv

**Reply by dzmikhalchuk** [reply]
@ASaiAnudeep what are the advantages of 'phin' now after switching to axios in [4.0](https://github.com/ethanent/phin/issues/91#issuecomment-2019062175) version?

**Reply by ASaiAnudeep** [reply]
So far, I don't see any clear benefits. But the package size is increased from **22.7kB** to **2.13MB**.

**Comment by yetzt** [comment]
phin has been unmaintained for a year now and it's repository had been depublished. you could use [phn](https://www.npmjs.com/package/phn) as a drop in replacement.
